android/src/main/java/com/google/android/cameraview/Camera1.java

```diff
@@ -1569,6 +1579,18 @@ private void stopMediaRecorder() {
         }
     }
 
+    private void pauseMediaRecorder() {
+        if (Build.VERSION.SDK_INT >= 24) {
+            mMediaRecorder.pause();
+        }
```

Comment:
what happen in old versions?

Reply:
Haven't tried in older version. But as per documentation pause and resume is available from API version 24 ( https://developer.android.com/reference/android/media/MediaRecorder#pause() ). 
It may throw an exception.

Reply:
pause and resume is still not working in API 29 above